webapi/tct-canvas-html5-tests/html5-canvas-app/data/x86-memo/canvas_colorsandstyles_createlineargradient_001_baseline.md5

```diff
@@ -0,0 +1 @@
+5fdc658cc7b9f789241cc98e69bb42d8
```

Comment:
You can ask for the baseline by weekly, that tests can be tested smoothly in next release. I will merge it as-is.